ansible/roles/system/tasks/main.yml

```diff
@@ -19,6 +19,20 @@
     msg: "Detected BerryBoot installation. Skipping some steps."
   when: is_berryboot
 
+- name: Disable overscan
+  lineinfile:
+    dest: /boot/config.txt
+    regexp: ^disable_overscan=
+    line: disable_overscan=1
+  when: not is_berryboot
+
+- name: Avoid on-screen warnings about power and temperature
```

Comment:
I just researched what this is. Doesn't it seem to be a good idea to keep it in? (I have never seen it in real life on a Pi1,2 or 3). If I understand it correctly, we would know that our power supply is shitty.....